### PR TITLE
PMM-4418 - Release Notes for PMM 1.17.2

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-447/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-447/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-447/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-447/

File: doc/source/release-notes/1.17.2.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+* :pmmbug:`4311`: Debian 10 Buster PMM Client support
+* :pmmbug:`2313`: Configure AMI images for a 100GB disk instead of a 16GB one
+* :pmmbug:`3500`: Update tooltips for the MySQL InnoDB Compression dashboard
+
+Fixed Bugs
+================================================================================
+
+* :pmmbug:`511`: Group all metrics on MongoDB Cluster Summary dashboard by the 'replset' label
+* :pmmbug:`2558`: Wrong time format in MySQL Table Statistics dashboard
+* :pmmbug:`3089`: Problem with RDS after docker way upgrade
```
Comment (rnovikovP) [inline doc/source/release-notes/1.17.2.rst]: let's move this to the top of the list

File: doc/source/release-notes/1.17.2.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+================================================================================
+Percona Monitoring and Management 1.17.2
+================================================================================
+
+:Date: October 23, 2019
+
+*PMM* (`Percona Monitoring and Management <https://www.percona.com/doc/percona-monitoring-and-management/index.html>`_) is a free and open-source platform for managing and monitoring |MySQL|, |MongoDB|, and |postgresql| performance. You can run *PMM* in your own environment for maximum security and reliability. It provides thorough time-based analysis for |MySQL|, |MongoDB|, and |postgresql| servers to ensure that your data works as efficiently as possible.
+
+One of the improvements in this release is a substantial size increase for the AMI images, which allows now to reserve space for about 10 instances with 30 days retention. Another improvement is the updated Tooltips for the MySQL InnoDB Compression dashboard by providing a description of what the graphs display, along with links to related documentation resources:
```
Comment (rnovikovP) [inline doc/source/release-notes/1.17.2.rst]: Note: this is for NEW installations only.  need to mentioned + link to documentation  how to increase if needed

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-447/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-447/

File: doc/source/release-notes/1.17.2.rst [hunk 3]
```diff
@@ -0,0 +1,71 @@
+.. _1.17.2:
+
+================================================================================
+Percona Monitoring and Management 1.17.2
+================================================================================
+
+:Date: October 23, 2019
```
Comment (rnovikovP) [inline doc/source/release-notes/1.17.2.rst]: October 24,

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-447/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-447/

**Pull request merged**